charts/DimensionWithData.ts

```diff
@@ -189,6 +194,14 @@ export class DimensionWithData {
         return this.variable.entities
     }
 
+    /*  returns [year of latest value, latest value] for a given entity and variable
```

Comment:
nit: I think you can remove this comment any rely on TypeScript's inference

Reply:
I did want to specify more than the type here, though. I wanted to specify what the values in the tuple represented. I could rename the function, but I'm afraid for clarity it would have to be yearAndValueOfLatestValueforEntity(). Alternatively, I could create local variables in the function called latestValue, latestYear, and return a tuple of those. What do you think is the best style?

Reply:
Ah, I get it. I'm fine with `yearAndValueOfLatestValueforEntity`.

Otherwise I'd vote for local variables over comments so the code is self-documenting. But it's a nit.

Tangential: I wonder if we should have type aliases "declare type owidValue = number` and `declare type owidYear = number`, which would indicate that they are ultimately coming from the DB. That might also be a helpful step in identifying all the places we need to update the code for generalizing year.

Reply:
> Ah, I get it. I'm fine with yearAndValueOfLatestValueforEntity.

Sounds good, I agree.

> Tangential

I like that idea -- maybe as oYear or something short if we do it across the codebase :)